app/templates/components/gh-theme-error-li.hbs

```diff
@@ -0,0 +1,13 @@
+<li>
```

Comment:
Not to be nitpicky -> but it seems like if we set `tagName = 'li'` in the component it would remove the need for this wrapping `li` element?